src/include/stir/KOSMAPOSL/KOSMAPOSLReconstruction.h

```diff
@@ -235,12 +235,12 @@ class KOSMAPOSLReconstruction:
 
   /*! Create a matrix similarly to calculate_norm_matrix() but this is done for the anatomical image, */
   /*! which does not  change over iteration.*/
-    void calculate_norm_const_matrix(TargetT &normm,
+    void calculate_norm_const_matrix(std::vector<shared_ptr<TargetT> > normm,
```

Comment:
this doesn't have any output apparently, but only side-effects? is that correct? best to document. Please make `normm` a reference, `const` if input`, non-const if output.